src/core/annotation.js

```diff
@@ -294,6 +294,27 @@ class AnnotationFactory {
       dependencies,
     };
   }
+
+  static async createNewAnnotationForPrinting(evaluator, task, annotations) {
```

Comment:
The method just above is called `saveNewAnnotations`, hence the following (and shorter) name seems more appropriate here:
```suggestion
  static async printNewAnnotations(evaluator, task, annotations) {
```